torch/_dynamo/variables/misc.py

```diff
@@ -658,11 +660,7 @@ def visit(node):
 
         VariableTracker.visit(visit, (args, kwargs))
 
-        if (
-            requires_grad
-            and torch.is_grad_enabled()
-            and config.capture_autograd_function
-        ):
+        if requires_grad and torch.is_grad_enabled():
```

Comment:
The flag is no longer participating in the logic here, but it seems like it has not actually been removed. Is it still used somewhere else? 

If not, and here is some downstream issue blocking actual removal, we should at least produce a warning on set if the config option is going to have no effect.